build.gradle

```diff
@@ -2,9 +2,13 @@
 buildscript {
   repositories {
     jcenter()
+    maven {
+      url 'https://maven.google.com/'
+      name 'Google'
+    }
```

Comment:
I guess it can be just `google()` as stated here: https://androidstudio.googleblog.com/2017/10/android-studio-31-canary-1-now-available.html
And it should be above `jcenter()`.